Code/GraphMol/MolDraw2D/DrawTextFTSVG.cpp

```diff
@@ -8,10 +8,16 @@
 // Original author: David Cosgrove (CozChemIx) on 08/05/2020.
 //
 
+#include <boost/format.hpp>
 #include <GraphMol/MolDraw2D/DrawTextFTSVG.h>
 
 namespace RDKit {
 
+namespace {
+std::string formatDouble(double val) {
```

Comment:
Since this function gets used multiple times I think it makes sense to define it `inline` in `MolDraw2DDetails.h`